packages/contracts-bedrock/foundry.toml

```diff
@@ -59,3 +59,13 @@ runs = 512
 
 [profile.lite]
 optimizer = false
+
+################################################################
+#                         PROFILE: KONTROL                     #
+################################################################
+
+[profile.kontrol]
+src = 'src/L1'
```

Comment:
Does this mean it will never be able to operate on contracts in other directories?

Reply:
As long as using the `kontrol` profile, no. But this is intended since the only contracts Kontrol cares about are in that folder. Including other folders would only bloat the Kontrol building stage and add extra noise. The reasoning behind this restriction is only to build what we need, but let me know if you have any further concerns

Reply:
Agreed this should be ok for now and I believe it will reduce runtime and artifact size since less is being compiled. Going to mark this resolved, since we can change this if we ever add proofs for other directories